docs/wiki/Verification_and_Validation_Plan.md

```diff
@@ -0,0 +1,133 @@
+# 1 Introduction
+This verification and validation plan applies to the NASA Operational Simulator (NOS).  NOS is described in more detail in the User's Manual and Developer's Guide and section 3.
```

Comment:
Do we have a user's manual and developer's guide? I think those are no longer maintained and we just have the readthedocs wiki.

Reply:
Removed the reference to user's manual and developer's guide.  Added reference to the Read the Docs Wiki.